storage/remote/queue_manager.go

```diff
@@ -20,13 +20,20 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/common/log"
 	"github.com/prometheus/common/model"
+	"github.com/prometheus/prometheus/config"
+	"github.com/prometheus/prometheus/relabel"
 )
 
 // String constants for instrumentation.
 const (
 	namespace = "prometheus"
 	subsystem = "remote_storage"
 	queue     = "queue"
+
+	defaultQueueCapacity     = 100 * 1024 / 10
```

Comment:
I just moved this around, but I'm sure the `10` is because of the default number of shards (10), while the `100` might be just arbitrary. @tomwilkie?